charts/spire/charts/spire-server/templates/service.yaml

```
@@ -31,31 +31,22 @@ spec:
 ---
 apiVersion: v1
 kind: Service
-metadata:
-  namespace: {{ include "spire-server.namespace" . }}
-  name: {{ include "spire-tornjak.frontend" . }}
-spec:
-  type: {{ .Values.service.type }} # ClusterIP
-  selector:
-    {{- include "spire-server.selectorLabels" . | nindent 4 }}
-  ports:
-    - name: {{ include "spire-tornjak.frontend" . }}
-      port: 3000
-      targetPort: 3000
-      protocol: TCP
----
-apiVersion: v1
-kind: Service
 metadata:
   namespace: {{ include "spire-server.namespace" . }}
   name: {{ include "spire-tornjak.backend" . }}
+  {{- with .Values.service.annotations }}
```

```suggestion
  {{- with .Values.tornjak.service.annotations }}
```

Shouldn't this service be annotated with tornjak specific annotations.